website/docs/docs/cloud/migration.md

```diff
@@ -41,6 +41,12 @@ These actions are required to prevent users from losing access <Constant name="c
 
     Keep the old <Constant name="cloud" /> IPs listed until the migration is complete.
 
+## Run related data retention
+
+All runs available in <Constant name="cloud"/> will be migrated with your account. This includes metadata about the run like its status, when it began executing, and the duration. The individual steps associated with a run will not be migrated with your account. Therefore, the <Constant name="dbt"/> commands executed during a run, along with their logs and artifact files, will not be available in <Constant name="cloud"/> after your migration.
```

Comment:
@aaronraff Should we add a note to clarify that run steps, logs, and artifacts will populate for net-new runs in the new instance? Or is that too in the weeds?

Reply:
How do you think we should word that? I originally omitted that information since that is the expected behavior during normal operation.